docs/01_introduction/README.md

```diff
@@ -143,60 +143,60 @@ Some commonly used optional easyconfig parameters include:
 * ``easyblock``, which specifies which (generic) easyblock should be used;
 * ``sources`` and ``source_urls``, which specify the list of source files and where to download them;
 * ``dependencies`` and ``builddependencies``, which specify (drum roll...) the list of (build) dependencies;
-* ``configopts``, ``buildopts``, and ``installopts``, which speficy options for the configuration/build/install commands, resp.;
+* ``configopts``, ``buildopts``, and ``installopts``, which specify options for the configuration/build/install commands, respectively;
+
+If these parameters are not provided, default values will be provided or extrapolated. For clarity, specifying at least
```

Comment:
"default values will be used"?

Reply:
"extrapolated and used"? I'm thinking the default values are not always the same default values...

Reply:
How about:

```
If these parameters are not provided, the corresponding default value will be used.
```

Reply:
Ack.